docs/src/utils/getStage.ts

```diff
@@ -0,0 +1,17 @@
+enum STAGE {
+  LOCAL = 'local',
+  DEV = 'dev',
+  PROD = 'prod',
+}
+
+export const getStage = () => {
+  const url = process.env.SITE_URL;
```

Comment:
can we have this fallback to an empty string so that the docs don't crash locally if your environment variables aren't set up?

Reply:
alternatively we can check for existence of url before using url.includes

Reply:
Done